project/BuildDependencies/scripts/0_package.list

```diff
@@ -7,8 +7,8 @@
 ; -> ...
 ;PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER!
 bzip2-1.0.5-win32.7z
-crossguid-8f399e8bd4-win32.7z
-curl-7.42.1-win32.7z
+crossguid-8f399e-win32-vc140.7z
+curl-7.46.0-win32-vc140.7z
```

Comment:
I was not sure if this one can be used because it has to linked statically (compared to the older one wich contains the dll) - Refering to the DLLCurl implementation kodi currently expects dynamic linkage...